### PR TITLE
Added first draft of README

### PR DESCRIPTION
**Pull request by jaxels10**
This is the first draft of the README for the organization.

### CODE REVIEW

**Comment by mikkelsiggaard** [comment]
@jaxels10 

Generally I think is pretty good. 

I do have some comments though:
1. If we are using a single license (e.g. MIT or Apache) I think it would be a good place to state that we publish all code under that license. If we (intend to) use multiple licenses we can state the most restrictive one or at least a bit of text stating our general commitment to Open Source. 
2. Instead of just writing "SCCP" we should write "Secure Compute and Communication Platform (SCCP)". 
3. Add a bit about why we are developing SCCP and perhaps mention RTP? 
4. Add an explanation about the term "distributed" and why this is important for the energy sector/Energinet.

**Comment by klausenbusk** [comment]
If we want this README file on the organization page it must be named `profile/README.md`.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# .github
```
Comment (klausenbusk) [inline README.md:1]: If we want this README file on the organization page it must be named profile/README.md.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+
+This Github organization was created to have a common place for our repositories that are developed open source and we welcome anyone to contribute or give feedback to our code. Since we are a team that works with many different technologies and works in different problem domains, we have created a [wiki](https://github.com/distributed-technologies/wiki) where you can get a full overview of the project we are working on, as well as find information about our development methods. 
+
+## License
+We publish all our code under the Apache 2.0 license. We use many different open source projects in this organization and we do not overrule their respective licenses. 
+
+## Projects
+
+#### Secure compute and communications platform(SCCP)
```
[the remark this answers is not in the document]
Comment (jaxels10) [reply README.md]: Added it

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+One of the projects we are developing is called SCCP, which is a highly available, fast, secure, robust platform which will run future applications for the energy grid in Denmark. We are developing this platform to enable other teams at Energinet to create algorithms that can run on the platform and in the future make decisions for the danish power grid. One instance of the platform that is being built at the moment is called the real-time platform(RTP). This platform runs Kafka and will be able to provide data from certain systems to algorithms if they request it and then generate an output. 
+If you are visiting this page to try out this platform and develop applications for it, please visit the [Yggdrasil repository](https://github.com/distributed-technologies/yggdrasil) to deploy our platform to Azure in a simple and fast way.
```
[the remark this answers is not in the document]
Comment (jaxels10) [reply README.md]: Implemented

File: README.md [hunk 4]
```diff
@@ -1,2 +1,16 @@
...
+
+## License
+We publish all our code under the Apache 2.0 license. We use many different open source projects in this organization and we do not overrule their respective licenses. 
+
+## Projects
+
+#### Secure compute and communications platform(SCCP)
+One of the projects we are developing is called SCCP, which is a highly available, fast, secure, robust platform which will run future applications for the energy grid in Denmark. We are developing this platform to enable other teams at Energinet to create algorithms that can run on the platform and in the future make decisions for the danish power grid. One instance of the platform that is being built at the moment is called the real-time platform(RTP). This platform runs Kafka and will be able to provide data from certain systems to algorithms if they request it and then generate an output.
```
[the remark this answers is not in the document]
Comment (jaxels10) [reply README.md]: Implemented

File: README.md (hunk 1 above)
Comment (jaxels10) [reply README.md:1]: Done

**Review by klausenbusk — DISMISSED** [review]
Two minor things, but LGTM 🎉

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+## License
+We publish all our code under the Apache 2.0 license. We use many different open source projects in this organization and we do not overrule their respective licenses. 
+
+## Projects
+
+#### Secure compute and communications platform (SCCP)
+One of the projects we are developing is called SCCP, which is a highly available, fast, secure, robust platform which will run future applications for the energy grid in Denmark. We are developing this platform to enable other teams at Energinet to create algorithms that can run on the platform and in the future make decisions for the danish power grid. One instance of the platform that is being built at the moment is called the real-time platform (RTP). This platform runs Kafka and will be able to provide data from certain systems to algorithms if they request it and then generate an output. 
+If you are visiting this page to try out this platform and develop applications for it, please visit the [Yggdrasil repository](https://github.com/distributed-technologies/yggdrasil) for instructions on how to deploy our platform to Azure in a simple and fast way.
```
Comment (klausenbusk) [inline profile/README.md:13]: I'm guessing you want a line break here?
```suggestion

If you are visiting this page to try out this platform and develop applications for it, please visit the [Yggdrasil repository](https://github.com/distributed-technologies/yggdrasil) for instructions on how to deploy our platform to Azure in a simple and fast way. 
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+## License
+We publish all our code under the Apache 2.0 license. We use many different open source projects in this organization and we do not overrule their respective licenses. 
+
+## Projects
+
+#### Secure compute and communications platform (SCCP)
+One of the projects we are developing is called SCCP, which is a highly available, fast, secure, robust platform which will run future applications for the energy grid in Denmark. We are developing this platform to enable other teams at Energinet to create algorithms that can run on the platform and in the future make decisions for the danish power grid. One instance of the platform that is being built at the moment is called the real-time platform (RTP). This platform runs Kafka and will be able to provide data from certain systems to algorithms if they request it and then generate an output.
```
Comment (klausenbusk) [inline profile/README.md:12]: This should be synced with @skovbakke changes in https://github.com/distributed-technologies/SCCP-documentation/pull/28 to https://github.com/distributed-technologies/SCCP-documentation/pull/27.

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+## License
+We publish all our code under the Apache 2.0 license. We use many different open source projects in this organization and we do not overrule their respective licenses. 
+
+## Projects
+These are the projects that we as team Distributed Technologies are currently working on. 
+
+#### Secure compute and communications platform (SCCP)
+The Secure Compute and Communication Platform (SCCP) project is an attempt at creating a modern, fast, secure, robust, and uniform platform for running a variety of application suites for running and monitoring the Danish energy grid. We are developing this platform to enable other teams at Energinet to create algorithms that can run on the platform and in the future make decisions for the danish power grid.
```
Comment (SomeFriendlyGhost) [inline profile/README.md:13]: do we need to tell what the platform is used for?

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

**Review by SomeFriendlyGhost — APPROVED** [review]
LGTM

**Pull request merged**